crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1747,6 +1747,24 @@ impl IgnoredSchedulingAmbiguities {
     pub fn allow_ambiguous_resource<T: Resource>(&mut self, world: &mut World) {
         self.0.insert(world.components.init_resource::<T>());
     }
+
+    /// Iterate through the [`ComponentId`]'s that will be ignored.
+    pub fn iter(&self) -> Iter<'_, ComponentId> {
```

Comment:
I would prefer `-> impl Iterator<Item = ComponentId> + '_` to avoid exposing implementation details.